server/sandbox_stop.go

```diff
@@ -24,10 +25,13 @@ func (s *Server) StopPodSandbox(ctx context.Context, req *pb.StopPodSandboxReque
 	if err != nil {
 		return nil, err
 	}
-
-	if err := s.netPlugin.TearDownPod(netnsPath, podNamespace, sb.id, podInfraContainer.Name()); err != nil {
-		return nil, fmt.Errorf("failed to destroy network for container %s in sandbox %s: %v",
-			podInfraContainer.Name(), sb.id, err)
+	if _, err := os.Stat(netnsPath); err == nil {
```

Comment:
yes, that's what my code does as well. If no error on stat then teardown, if error other that not exists, return, otherwise continue the stop routine to cleanup other resources.